lib/spack/spack/test/concretize.py

```diff
@@ -1357,14 +1357,15 @@ def test_multivalued_variants_from_cli(self, spec_str, expected_dict):
             # Version 1.1.0 is deprecated and should not be selected, unless we
             # explicitly asked for that
             ("deprecated-versions", ["deprecated-versions@1.0.0"]),
-            ("deprecated-versions@1.1.0", ["deprecated-versions@1.1.0"]),
+            ("deprecated-versions@=1.1.0", ["deprecated-versions@1.1.0"]),
```

Comment:
can you drop this array, makes little sense

Reply:
Sure, I'll do it. It's really nitpick though (and not changed by this PR).

Reply:
Done

Reply:
Takes a moment to figure out what the test does otherwise :man_shrugging: